.travis.yml

```diff
@@ -24,6 +24,8 @@ branches:
 
 matrix:
   fast_finish: true
+  allow_failures:
+    - env: SYMFONY_VERSION='3.0.*@dev'
```

Comment:
should be removed once https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1188 is merged